src/components/Geogebra/GeogebraViews.js

```diff
@@ -74,15 +75,43 @@ export default class {
             }
         });
 
+        api.service('elements').on('removed', (element) => {
+            const pos = this.workshopIds.indexOf(element.workshop);
+            if (pos !== -1) {
+                this.GAVs[pos].deleteObject(element.name);
+            }
+        });
+
         api.service('workshops').on('xml-changed', (workshop) => {
             const pos = this.workshopIds.indexOf(workshop.id);
```

Comment:
Ditto

Reply:
I need the position in array, not only Boolean.

Reply:
I need the position in array, not only Boolean.